docs/_themes/clusterhq/static/pygments.css

```diff
@@ -0,0 +1,135 @@
+.highlight code,.highlight pre {
+color:#fdce93;
```

Comment:
For the sake of whoever has to maintain this, please add comments to this file - you choose what, just whatever you'd like to see if you wanted to modify the theme and didn't remember this file.